tests/lib/rules/no-unsupported-features/node-builtins.js

```diff
@@ -5505,7 +5220,7 @@ new RuleTester({ languageOptions: { sourceType: "module" } }).run(
                             messageId: "unsupported",
                             data: {
                                 name: "vm.Module",
-                                supported: "9.6.0",
+                                supported: "(none yet)",
```

Comment:
What's the meaning of "not yet"? experimental?

Reply:
It means that its not stable in node yet :)
https://nodejs.org/api/vm.html#class-vmmodule

Reply:
so the final message is "The "vm.Module" is not supported until Node.js (not yet). The configured version range is '{{version}}'."

https://github.com/eslint-community/eslint-plugin-n/blob/4b12cdc87f7f8daf7045d9b5ff72cbb33bf67d49/lib/util/check-unsupported-builtins.js#L115

maybe we could add a new messageId for this?

Reply:
mmm, that sounds like a nicer option!

Reply:
Resovled in [`a0159d5` (#180)](https://github.com/eslint-community/eslint-plugin-n/pull/180/commits/a0159d50ef207d74258bcbf0966b0e4c4c21162a)